src/iced/top.iced

```diff
@@ -47,6 +48,7 @@ exports.run = run = ({passphrase, salt, progress_hook}, cb) ->
 
   out = generate seed_final
   out.seeds = seeds
+  out.mnemonic = mnemonic.fromSeed(seed_final.slice(0,16), mnemonic.Words.ENGLISH).toString()
```

Comment:
The added extra entropy by the stretching is insignificant.

Thinking in bits is hard, but to make things simple to understand, let's imagine the user had a super strong password. 80 bits (which would be a 25 character password completely random from a 70 character set)

In order for key stretching to provide equivalent 48 bits of the remaining neede effective entropy, it would need to make the calculations be 2.81 x 10^14 times more expensive in calculation time.

If the original calculation took one microsecond, your stretching would need to take 8 years to make 48 bits of effective entropy.

tl;dr Key stretching is NOT meant to replace entropy. And entropy is a LOT harder to gather than people think.